src/Model/ModelManager.php

```diff
@@ -337,6 +337,12 @@ public function getIdentifierValues($entity)
                 continue;
             }
 
+            if (method_exists($value, '__toString')) {
```

Comment:
this should be `\method_exists`, FlintCI slacking again.

Reply:
Agree, I will update it now